src/view/treeNodes/categoryNode.ts

```diff
@@ -184,7 +184,7 @@ export class CategoryTreeNode extends TreeNode implements vscode.TreeItem {
 				}
 			} else {
 				try {
-					const response = await this._prManager.getPullRequests(this._type, { fetchNextPage: true });
+					const response = await this._prManager.getPullRequests(this._type, { fetchNextPage: true }, this._categoryQuery);
```

Comment:
This was missing previously, causing "fetch more" on "created by me" (for instance), to just load the next page of "all"